docs/modules/pipelines/pages/configuring-jobs.adoc

```diff
@@ -171,7 +171,15 @@ cluster change depending on these two settings.
 
 The following settings allow you to configure whether Hazelcast stores metrics such as the time that a job started and completed.
 
-NOTE: These metrics are not available in SQL. To access these metrics, you must use the Jet API.
+[NOTE]
+====
+These metrics are not available in SQL. To access them, you must use the Jet API's `JobConfig()` object. For example:
+
+[source,java]
+----
+JobConfig.setStoreMetricsAfterJobCompletion(true);
+----
+====
```

Comment:
```suggestion
These metrics are not available in SQL. To access these metrics, you must use the Jet API:

[source,java]
----
JobMetrics metrics = job.getMetrics();
----
====
```

Unless you enable `storeMetricsAfterJobCompletion`, metrics are available only for running jobs.